scripts/jupyter_dask_snellius.bsh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --cpus-per-task=2  # minimum allocation on Snellius is 32 cores
```

Comment:
Maybe use here 32, since anyway one is billed by this amount?

```suggestion
#SBATCH --cpus-per-task=32  # minimum allocation on Snellius is 32 cores
```